nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -25,16 +26,21 @@ contract PolicyManager is Upgradeable {
 
     event PolicyCreated(
         bytes16 indexed policyId,
-        address indexed client
+        address indexed creator,
```

Comment:
Rather than `creator`, it should be something like `sponsor`, `payer`, etc.

Reply:
✔️